Multisig/Data/Services/Utils/GSError.swift

```diff
@@ -534,6 +534,15 @@ enum GSError {
         let loggable = true
     }
 
+    struct SecureStoreAccessError: DetailedLocalizedError {
```

Comment:
if it's canceled by user, then it's not an access error.

Reply:
Let's rethink about that as part of error handling